### ISSUE TITLE
Embedding engine does not like RTX A4000 16GB GPU

### ISSUE THREAD

**Issue by tbendien** [bionic-gpt/bionic-gpt#301]
I am getting an error while trying to chat with a document. Logs are attached below. Obviously the context size is an issue, but we can deal with that later.

The thing I am worried about is thatI put the raw logs into Claude and it said that one of the issues is the embeddings-api_ is failing to start because of a GPU incompatibility.

Obviously this is a significant issue, as I have a bunch of RTX A4000 16GB GPU's which are not that old which I want to use, but now may not be compatible with the embeddings engine......

Claude response: 

Based on the logs, there seem to be a few issues:

The text generation server (tgi_1) is running into errors generating text, likely due to the input being too long. It is failing with an error that the total input length + max_new_tokens exceeds the limit of 2048 tokens.

The embeddings API server (embeddings-api_1) is failing to start with an error about incompatible GPU compute capability. It seems the server code is compiled for a newer GPU architecture than the actual GPU supports.

There are no obvious issues with the application server (app_1) or the LLMs API server (llm-api_1). They seem to be operating normally.

The OAuth2 proxy and Postgres database servers also seem to be running without issues.

So in summary, the main problems seem to be with the text generation server hitting token limits on inputs and failing to generate text, as well as the embeddings server failing to start due to GPU incompatibility. Fixing these issues would allow the system to operate more smoothly.

[Sample Log Files - Bionic GPU too old.pdf](https://github.com/bionic-gpt/bionic-gpt/files/13851660/Sample.Log.Files.-.Bionic.GPU.too.old.pdf)

**Comment by tbendien** [comment]
Whoops, I was using an older P2200 5GB GPU in this machine, the RTX A4000 works just fine :-)
